packages/events-example/src/commands/reconcile/index.ts

```diff
@@ -13,6 +13,7 @@ type NileConfig = {
 type DeveloperCreds = {
   email: string; // developer email
   password: string; // developer password
+  devtoken: string; // developer access token
 };
```

Comment:
These `DeveloperCreds` are going away in the `nile-agent` version of this. But most of the rest of this PR should transfer over.

Reply:
where `DeveloperCreds` are now, `devtoken` should not be added there.